src/widget/OktaSignIn.js

```diff
@@ -216,6 +216,7 @@ var OktaSignIn = (function () {
       headers: {
         'X-Okta-User-Agent-Extended': 'okta-signin-widget-' + config.version
       },
+      tokenManager: options.tokenManager,
```

Comment:
Is this PR out of date with master? The SIW currently accepts ALL options under `authParams` and passes them to `OktaAuth`. This includes the `TokenManager` section. https://github.com/okta/okta-signin-widget/blob/88c158ef70b1c2e2323264d5e2c136550320675f/src/widget/OktaSignIn.js#L148

Reply:
Yeah I figured that out afterwards by trawling the code and it works! Thanks. I would suggest putting this is in the docs rather than update this PR